tests/assets/a1.json

```
@@ -11,7 +11,7 @@
             "en-US":"created"
         }
     },
-    "object":{
+    "statementobject":{
```

Again, I think all these references in the assets need to be changed back.
```suggestion
    "object":{
```

---

see #18